app/services/i_scream_service.rb

```diff
@@ -1,8 +1,12 @@
 class IScreamService 
 
   def conn 
-    # Faraday.new('http://localhost:9393')
-    Faraday.new('https://i-scream-microservice.herokuapp.com')
+    # Faraday.new('http://localhost:9393') do |faraday|
+
+    Faraday.new('https://i-scream-microservice.herokuapp.com') do |faraday|
+      faraday.options[:timeout] = 300
```

Comment:
API is so slow we had to increase the timeout settings :/

Reply:
:( but for our eval demo, we should just use the same searches and do what ian suggested and just cache it